standup.md

```diff
@@ -0,0 +1,6 @@
+# Standup
+
+Standup will be conducted everyday starting ``10:30am`` Monday to Friday. The venue for the virtual Standup will be conducted over [Google Meet](https://meet.google.com/sjr-pjar-mjq). 
```

Comment:
Just a comment: the url might change for some reason (like scheduling a new one next month). I've explored it in google calendar, and I think we can personalize the url to be recognizable than an auto-generated one. 

Not sure, but mentioned it anyway as we might need to be update this every time if it is indeed the case.

Reply:
Oops. checked it again. one can only enter an existing meeting code but still i'm not sure. But i guess this won't change.